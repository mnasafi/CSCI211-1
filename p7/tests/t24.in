echo t24: Insert days of week into tree, then call balanced, rebalance, balanced, print
insert Tuesday
insert Thursday
insert Wednesday
insert Sunday
insert Saturday
insert Monday
insert Friday
breadth
balanced
echo calling rebalance
rebalance
breadth
balanced
