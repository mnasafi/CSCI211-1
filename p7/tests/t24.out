t24: Insert days of week into tree, then call balanced, rebalance, balanced, print
{Tuesday, Thursday, Wednesday, Sunday, Saturday, Monday, Friday}
Tree is not balanced.
calling rebalance
{Sunday, Monday, Tuesday, Friday, Saturday, Thursday, Wednesday}
Tree is balanced.
